src/csharp/Grpc.Tools.Tests/ProtoCompileCommandLineGeneratorTest.cs

```diff
@@ -140,6 +140,15 @@ public void GenerateGrpcWithOptions()
                         Does.Contain("--grpc_opt=baz,quux"));
         }
 
+        [Test]
+        public void AdditionalProtocOptions()
```

Comment:
nit: rename the test method name to AdditionProtocArguments as well

Reply:
Good catch! Fixed